com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -51,32 +78,55 @@ public enum ShaderOptions
         // Edit->Render Pipeline->Generate Shader Includes
         // Probe Volumes feature must also be enabled inside of your HDRenderPipelineAsset.
         // Also uncomment in the HDRP package all ".../Experimental/Probe Volume" menu
+
+        /// <summary>Probe volume evaluation mode.</summary>
```

Comment:
The probe volume evaluation mode.
Can you also add:
\<seealso cref="ProbeVolumesEvaluationMode "/>